The test succeeds although it seems to fail in some of the upstream CI configs.
Just disable it.

Author: Alexander Grund (TU Dresden)

--- a/test/test_dataloader.py
+++ b/test/test_dataloader.py
@@ -1388,7 +1388,7 @@ except RuntimeError as e:
     # please don't forget to remove this skip when remove the xfailIfLinux.
     @skipIfXpu
     # https://github.com/pytorch/pytorch/issues/128551
-    @xfailIfLinux
+    @unittest.skip("unexpected success on e.g. AMD CPU")
     def test_segfault(self):
         p = ErrorTrackingProcess(target=_test_segfault)
         p.start()
